lib/safe-pg-migrations/plugins/idem_potent_statements.rb

```diff
@@ -11,52 +12,62 @@ def add_index(table_name, column_name, **options)
       remove_index(table_name, name: index_name)
       super
     end
+    ruby2_keywords :add_index if respond_to?(:ruby2_keywords, true)
 
-    def add_column(table_name, column_name, type, options = {})
+    def add_column(table_name, column_name, type, *)
```

Comment:
I think here, we should stick to the Rails method signature `def add_column(table_name, column_name, type, **options)` and it will just work fine. Even without `ruby2_keywords :add_index if respond_to?(:ruby2_keywords, true)`